src/components/Notifications/NotificationFeed/NotificationFeed.js

```diff
@@ -262,7 +265,9 @@ class NotificationFeed extends React.Component {
     fetchNotificationFeed({
       ...getAuthentication(this.props.authentication),
       offset: 0,
-      limit,
+      limit: this.props.enableFetchAllNotificationFeedAcrossProjects ? 10 : limit,
```

Comment:
Could add this condition to where the limit variable is being defined above, so that all the logic for this `limit` field is in one spot/line

Reply:
sure, good caught, thx mate!